readthedocs/notifications/messages.py

```diff
@@ -235,6 +222,7 @@ def get_rendered_body(self):
             textwrap.dedent(
                 """
             Build output directory for format "{{artifact_type}}" is not a directory.
+            Make sure you created this directory properly when running <code>build.commands</code>.
```

Comment:
I think we should leave mentions of build.commands out, as this is true for all build workflows

```suggestion
            Make sure this directory is properly created and any necessary files are created in the directory.
```

Reply:
Good point!